components/storage/blockdevice/COMPONENT_QSPIF/mbed_lib.json

```diff
@@ -0,0 +1,20 @@
+{
+"name": "qspif",
+    "config": {
+        "QSPI_FREQ": "40000000"
```

Comment:
In my comment above I notice you do not use this default in the constructor. I found out that SPIFBlockDevice has the same bug. Please fix it here, I'll take care of the SPIF one.

Reply:
Fixed - thanks